tests/unit/test_rust_backend.py

```
@@ -0,0 +1,519 @@
+            try:
+                with patch('subprocess.run'):
+                    backend = RustBackend(config)
+                    assert backend is not None
+            except (ValueError, TypeError, RustBackendError):
+                pass
+
+
+def test_module_imports():
+    """Test that all required modules can be imported."""
+    try:
+        import subprocess
```

Ruff (0.11.9)

`subprocess` imported but unused; consider using `importlib.util.find_spec` to test for availability

(F401)